### PR TITLE
# Kata Containers 1.3.0-rc1

### PR DESCRIPTION
**Pull request by jcvenegas**
- build: Fix systemd unit file build error
- trigger: Fix virtcontainers paths

3a3c5a4 build: Fix systemd unit file build error
c7eb9f3 build: Add kata prefix to throttler name
0a321f3 build: Add unit files build target
0ca55b4 trigger: Make vc throttler Kata-specific
61fa8d5 comments: Don't hard-code generated path
7926668 throttler: Make socket path Kata-specific
a4a7e6a paths: Fix throttler socket path
e17d230 trigger: Fix virtcontainers paths
ae46e77 build: Use wildcard make function
67ec1cc build: Remove duplicate rule for service files

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.2.0
+1.3.0-rc1
```
Comment (amshinde) [inline VERSION]: Did we miss the 1.3.0-rc0 release on this one?

Comment (jcvenegas) [reply VERSION]: Looks like we did , we are not bumping for stable releases only tagging, this repository should not break kata so we only maintain one branch and bump when kata in master has a new release.

Comment (jodh-intel) [reply VERSION]: We could therefore add the `-rc0` tag at the correct commit now right?

**Comment by jodh-intel** [comment]
lgtm

File: VERSION (hunk 1 above)
Comment (egernst) [reply VERSION]: agreed @jodh-intel

Comment (jodh-intel) [reply VERSION]: Thanks for adding it @jcvenegas!

**Pull request merged**